WalletWasabi.Gui/Tabs/WalletManager/LoadWalletViewModel.cs

```diff
@@ -629,9 +630,12 @@ public async Task LoadWalletAsync()
 					return;
 				}
 
+				var walletManager = Global.WalletManager;
+				WalletService walletService = null;
 				try
 				{
-					var walletService = await Task.Run(async () => await Global.CreateWalletServiceAsync(keyManager));
+					walletService = await Task.Run(async () => await walletManager.AddWalletServiceAsync(keyManager));
+					await walletManager.StartWalletServiceAsync(walletService, CancellationToken.None);
```

Comment:
Does this mean we have no cancellation now?

Probably it doesnt matter?

Or to be fixed in future PR?

Reply:
This is where we will need the per wallet cancellationToken. This PR is already L so no more space to refactor that. In future PR.

Reply:
This is bad. It ruins and important functionality. I cannot cancel wallet load anymore.